Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -552,6 +559,28 @@ function uninstallHuntress {
     }
 }
 
+
+
+
+# grab the currently installed agent version AB
+function getAgentVersion {
+    $exeAgentPath = Join-Path (getAgentPath) "HuntressAgent.exe"
+    $agentVersion = (Get-Item $exeAgentPath).VersionInfo.FileVersion
+    LogMessage "Agent version $($agentVersion) found"
+    return $agentVersion
+}
+
+# ensure all the Huntress services are running and set them to delayed start AB
+function repairAgent {
+    sc.exe stop HuntressAgent 
```

Comment:
@Alan-Huntress instead of calling out to `sc.exe stop`, you can simply use `Service-Stop HuntressUpdater` and `Service-Start HuntressUpdater` (built in powershell commands)

Reply:
another note @Alan-Huntress don't stop the services at all - just start them. Using `Service-Start` in powershell will complete with no errors even if the service is already running.